tslint.json

```diff
@@ -0,0 +1,25 @@
+{
+  "extends": "tslint:recommended",
+  "rules": {
+    "quotemark": [
+      "single",
+      "avoid-escape"
+    ],
+    "no-var-requires": false,
+    "arrow-parens": false,
+    "only-arrow-functions": [
+      false
+    ],
+    "member-access": false,
+    "member-ordering": [
+      "field-first"
+    ],
+    // TODO: these should be activated at one point
```

Comment:
We should have more dignity than these TSLint barbarians. 😁 There are no comments allowed in JSON (https://jsonlint.com/ will show the truth). Let's be professional programmers who don't misuse well-defined standards. 🌵